src/a01/config.py

```diff
@@ -22,22 +22,6 @@ def check_environment():
                           'az account set -s 6b085460-5f21-477e-ba44-1035046e9101',
                           'The current az login failed to sign in '
                           '6b085460-5f21-477e-ba44-1035046e9101 subscription')
-    result &= verify_item('Azure Container Registry login',
-                          f'az acr login -n {DROID_CONTAINER_REGISTRY}',
-                          'The current az account must allow you to login container '
-                          f'registry {DROID_CONTAINER_REGISTRY}.')
-    result &= verify_item('Azure Container Service login',
```

Comment:
Is this going away because it is hardcoded?

Reply:
It is because today the CLI should not be dealing with the container registry anymore. All the action regarding registry is happening on server.